sources/packages/web/src/components/students/applicationTracker/Cancelled.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <application-status-tracker-banner
+    label="You've cancelled your application"
+    icon="fa:fas fa-exclamation-circle"
+    icon-color="danger"
+    background-color="error_bg"
+    ><template #content
+      ><span
```

Comment:
Is the span needed here?

Reply:
mm, span is generally used as a content container. that's  the reason i always use span for contents